includes/reports/data/customers/class-top-five-customers-list-table.php

```diff
@@ -125,6 +134,32 @@ public function get_columns() {
 		return $columns;
 	}
 
+	/**
+	 * Overrides the `spent` column value to possibly display in the filtered currency.
+	 *
+	 * @since 3.0
+	 *
+	 * @param array  $item
+	 * @param string $column_name
+	 *
+	 * @return string
+	 */
+	public function column_default( $item, $column_name ) {
+		if ( 'spent' !== $column_name ) {
+			return parent::column_default( $item, $column_name );
+		}
+
+		$currency          = '';
+		$selected_currency = Reports\get_filter_value( 'currencies' );
+		if ( ! empty( $selected_currency ) && 'convert' !== $selected_currency ) {
+			$currency = $selected_currency;
+		}
+
+		$value = edd_currency_filter( edd_format_amount( $item[ $column_name ] ), $currency );
+
+		return apply_filters( 'edd_customers_column_' . $column_name, $value, $item['id'] );
```

Comment:
If the `$column_name` will be `spent` here (as I read the conditional at line 148 it seems that the rest of the code is running only if it is `spent`, can the return line instead be:

```suggestion
		return apply_filters( 'edd_customers_column_spent', $value, $item['id'] );
```

I like dynamic filters but it feels like this really isn't one and it would be easier to search for it.

Reply:
Although you're right, I'm kind of inclined to leave it as is. The reason being that this filter is copied from the base class - `EDD_Customer_Reports_Table` which has this same dynamic filter, which is applied to all columns. I like keeping it this way to better show that they are actually the same filter, if that makes sense.